backend/api/serializers.py

```diff
@@ -0,0 +1,278 @@
+from django.contrib.auth import get_user_model
+from django.contrib.contenttypes.models import ContentType
+from rest_framework import serializers
+
+from api.models import (AttendanceStatus, Comment, Event, FollowStatus, Media,
+                        Tag, VoteStatus)
+
+
+class AttendanceCreateSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = AttendanceStatus
+        fields = ('id', 'event', 'status')
+
+    def create(self, validated_data):
+        user = self.context.get("request").user
+        attendance_status, created = AttendanceStatus.objects.update_or_create(
+            owner=user, event=validated_data['event'], defaults={'status':validated_data['status']})
```

Comment:
This may raise `KeyError`, which causes an unintended Http500 error. It's better using `.get(item)` function and handle it if it's not sent.